PhotoMap/Features/Map/Coordinators/ImagePickerCoordinator.swift

```diff
@@ -42,10 +42,10 @@ extension ImagePickerCoordinator: UIImagePickerControllerDelegate, UINavigationC
     }
 
     func imagePickerController(_ picker: UIImagePickerController,
-                               didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+                               didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]) {
         picker.dismiss(animated: true, completion: nil)
 
-        guard let image = info[UIImagePickerController.InfoKey.originalImage] as? UIImage else { return }
+        guard let image = info[UIImagePickerController.InfoKey.editedImage] as? UIImage else { return }
```

Comment:
I think you can also disable editing using `imagePicker.allowsEditing = false` and then you could use original image. Give it a try